modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,109 @@
+
+* With this update, users can implement an alternative sharding algorithm that utilizes consistent hashing with bounded loads. The algorithm ensures a uniform distribution of managed clusters across shards while minimizing the need for reshuffling during configuration changes, such as adjustments in the number of managed clusters or shards. link:https://issues.redhat.com/browse/GITOPS-3584[GITOPS-3584]
+
+* With this update, users can expose Argo CD Application labels on Kubernetes events generated by Argo CD. When events are created for applications that have specific label keys defined in the `resource.includeEventLabelKeys` field within the `argocd-cm` ConfigMap, the controller associates the relevant labels with these events. This enhancement facilitates the filtering and processing of events according to application labels. To set the `resource.includeEventLabelKeys` field in a {gitops-shortname} environment, use the `.spec.extraConfig` field in the Argo CD CR. The {gitops-shortname} Operator automatically incorporates these fields from the `.spec.extraConfig` field into the `argocd-cm` ConfigMap. link:https://issues.redhat.com/browse/GITOPS-3233[GITOPS-3233]
++
+.Example 
+[source,YAML]
+----
+```yaml
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-sample # <1>
+  namespace: default # <2>
+spec:
+  extraConfig:
+    resource.includeEventLabelKeys: team,env*
+```
+----
+<1> Specifies the name of the Argo CD Application label.
+<2> Specifies the namespace where the Argo CD Application label is defined.
++
+For more information, refer to link:https://argo-cd.readthedocs.io/en/latest/operator-manual/declarative-setup/#labels-on-application-events[Labels on Application Events].
+
+[id="fixed-issues-1-14-0_{context}"]
+== Fixed issues
+
+* Before this update, users could develop applications without the necessity of verifying commit signatures through the `ApplicationSet` CRD. This update fixes the issue because users can now verify commit signatures for `ApplicationSet` CDR created using Git generators. link:https://issues.redhat.com/browse/GITOPS-3049[GITOPS-3049]
++
+[NOTE]
+====
+The signature verification does not function with the templated project field in `ApplicationSet` CRD Git Generator.
+====
+
+* Before this update, users could not execute a rollback for applications with multiple sources in the Argo CD Web UI or CLI. This updates fixes the issues because users can now carry out rollbacks for applications using the Argo CD Web UI and CLI. link:https://issues.redhat.com/browse/GITOPS-3996[GITOPS-3996]
```

Comment:
Hi @ishitasequeira, since this RN entry is not a new feature I have decided to include only the first two entries for the multi-source applications in the New Features section.